kroma-chain-ops/genesis/testdata/test-deploy-config-devnet-l1.json

```diff
@@ -53,5 +52,25 @@
   "colosseumSegmentsLengths": "2,2,3,3",
   "zkVerifierHashScalar": "0xb54142d3712f03ff5c15914a3dce82c0d8a8d537e165e2342048adf14e34aa",
   "zkVerifierM56Px": "0x175a2d1aaf07ee184a955e35f3deac123ef1088b68c38bace37056dcfc9d304d",
-  "zkVerifierM56Py": "0x150856c3d1750c2578b823cb3c7d74ed3e94c1cfece7e1bfdaaff36eea788fb9"
+  "zkVerifierM56Py": "0x150856c3d1750c2578b823cb3c7d74ed3e94c1cfece7e1bfdaaff36eea788fb9",
+  "l1GovernanceTokenProxy": "0x4200000000000000000000000000000000000064",
+  "mintManagerOwner": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "l2MintManagerRecipients": [
```

Comment:
Let's add `l1MintManagerRecipients` and `l1MintManagerShares`.

Reply:
Since this file is only used to test build l2 genesis build, those configs are unnecessary. This file does not contain all deploy configs like `test-deploy-config-full.json`.